tei-py/python/structs.py

```
@@ -187,12 +199,19 @@ class DivBlock(msgspec.Struct, tag="div", tag_field="type", omit_defaults=True):
     such as chaptered material or show notes.
     """
     div_type: str
+    subtype: str | None = None
+    head: Head | None = None
     content: list[DivContent] = msgspec.field(default_factory=list)
     xml_id: str | None = None
 
     def __post_init__(self) -> None:
         if not self.div_type.strip():
             raise ValueError("DivBlock.div_type must contain non-whitespace text")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)